### DISCUSSION TITLE
Secret Scanning - Enable All for Push Protection states no repos are eligible nor enabled

### DISCUSSION THREAD

**Discussion by mbroz2** [community/community#57637] [Code Security]
### Select Topic Area

Bug

### Body

When Secret Scanning and Push protection are enabled (checked in the UI), if I then click 'Enable All' under Push protection it states that none of the repos are enabled nor eligible.  I thought it might a data propagation issue, but 12+ hours later, nothing has changed.
![image](https://github.com/community/community/assets/3076261/2c157a28-779d-4d34-b092-d7518201992e)

However, when navigating directly to the repos and checking their settings, the options are correctly enabled and the secret scanning and push protection is indeed working (I have verified).

I'm not sure if I'm misunderstanding what 'Enable All' is intended to do there and what the resulting popup is showing, but it seems like a bug to me.

**Comment by aashah** [comment]
Hey @mbroz2, sorry for the late answer 😅. Is this still happening?

**Comment by mbroz2** [comment]
yes, the UI has changed, but it's still reporting incorrect data:
![image](https://github.com/community/community/assets/3076261/83d49bc3-df81-4cb3-b35a-05c0bfec47e4)

**Reply by silva22322** [reply]
@bushuevkonstantun1408

**Reply by aashah** [reply]
I spoke with a few folks, and this dialog is really there to support organizations that have purchased advanced security as it gives them a much more clear sense of where licenses are being used with respect to billing. I think for free organizations, this doesn't carry over as much, and we'll likely remove some of the text in this dialog.
